wiz-admission-controller/templates/_helpers.tpl

```diff
@@ -189,3 +214,29 @@ Use for debug purpose only.
 {{- define "helpers.var_dump" -}}
 {{- . | mustToPrettyJson | printf "\nThe JSON output of the dumped var is: \n%s" | fail }}
 {{- end -}}
+
+{{- define "wiz-admission-controller.resources" -}}
+{{- if not .Values.hpa.enabled }}
+{{- if hasKey .Values "resources" }}
+{{- toYaml .Values.resources }}
+{{- else -}}
+{}
+{{- end -}}
+{{- else }}
+{{- if hasKey .Values "resources" }}
+{{- toYaml .Values.resources }}
+{{- else -}}
+requests:
+    cpu: 500m
+    memory: 300Mi
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "wiz-admission-controller.isEnforcerEnabled" -}}
```

Comment:
Nice! Very useful. You don't have to do this in this PR, but it can also be nice to use it in `service.yaml `and the secret in `opawebhook.yaml`.

Reply:
Done